bin/reth/src/node/events.rs

```diff
@@ -109,6 +113,23 @@ impl NodeState {
             }
         }
     }
+
+    fn handle_consensus_layer_health_event(&self, event: ConsensusLayerHealthEvent) {
```

Comment:
I like this, 

we could the message as  `ConsensusLayerHealthEvent::as_str: &'static str` function which would improve reusability but perhaps not really worth it

Reply:
I think same applies to `BeaconConsensusEngineEvent`. Also, `ConsensusLayerHealthEvent` is currently in the `bin/reth` package, so it wouldn't really improve reusability imo.